static/src/js/components/Map/ShapefileLayer.js

```diff
@@ -405,40 +410,81 @@ class ShapefileLayerExtended extends L.Layer {
   }
 }
 
-class ShapefileLayer extends MapLayer {
-  createLeafletElement(props) {
-    return new ShapefileLayerExtended(props)
-  }
+// class ShapefileLayer extends MapLayer {
+//   createLeafletElement(props) {
+//     return new ShapefileLayerExtended(props)
+//   }
+
+//   updateLeafletElement(fromProps, toProps) {
+//     const element = this.leafletElement
+
+//     const {
+//       isProjectPage: toIsProjectPage,
+//       leaflet: toLeaflet,
+//       shapefile: toShapefile
+//     } = toProps
+
+//     const { map } = toLeaflet
+
+//     element.map = map
+//     element.isProjectPage = toIsProjectPage
+
+//     const {
+//       file: toFile,
+//       shapefileId: toShapefileId,
+//       shapefileName,
+//       selectedFeatures
+//     } = toShapefile
+
+//     if (toFile) {
+//       element.drawNewShapefile(toFile, selectedFeatures)
+//     }
+
+//     if (!toShapefileId && !shapefileName) {
+//       element.onRemovedFile()
+//     }
+//   }
+// }
+
+// export default withLeaflet(ShapefileLayer)
```

Comment:
Commented out section should be removed, or marked as being renabled

Reply:
Removed